test/analysis/test_summary.py

```diff
@@ -0,0 +1,31 @@
+# Run from SPRAS root with python -m test.analysis.test_summary
+
+import pandas as pd
```

Comment:
I'll first run these tests locally and then help convert them to pytest test cases.